docs/api/remote-procedure-calls-raw-transactions.md

```diff
@@ -938,8 +938,8 @@ _Result (if verbose=`true`)---the decoded transaction_
 | →<br>`extraPayload`         | string (hex)   | Optional<br>(0 or 1)    | _Added in Dash Core 0.13.0.0_<br><br>Hex encoded DIP2 extra payload data. Only present if it's a DIP2 special transaction                                                                                                                                                                                                        |
 | →<br>`hex`                  | string (hex)   | Required<br>(exactly 1) | The serialized, hex-encoded data for the provided `txid`                                                                                                                                                                                                                                                                         |
 | →<br>`blockhash`            | string (hex)   | Optional<br>(0 or 1)    | If the transaction has been included in a block on the local best block chain, this is the hash of that block encoded as hex in RPC byte order                                                                                                                                                                                   |
-| →<br>`height`               | number (int)   | Optional<br>(0 or 1)    | The block height where the transaction was mined                                                                                                                                                                                                                                                                                 |
-| →<br>`confirmations`        | number (int)   | Required<br>(exactly 1) | If the transaction has been included in a block on the local best block chain, this is how many confirmations it has.  Otherwise, this is `0`                                                                                                                                                                                    |
+| →<br>`height`               | number (int)   | Optional<br>(0 or 1)    |  If the transaction has been included in a block on the local best block chain, this is the block height where the transaction was mined.  Otherwise, this is `-1`. Not shown for mempool transactions. |
```

Comment:
```suggestion
| →<br>`height`               | number (int)   | Optional<br>(0 or 1)    |  If the transaction has been included in a block on the local active block chain, this is the block height where the transaction was mined.  Otherwise, this is `-1`. Not shown for mempool transactions. |
```
But not sure. Your call